practice_app/api/tests.py

```diff
@@ -1,3 +1,50 @@
-from django.test import TestCase
-
+from django.test import TestCase,Client
+from unittest import skip
+import requests
+from . import api_keys
 # Create your tests here.
+class google_scholar_test_cases(TestCase):
```

Comment:
Hey, I was just looking at this code again and I noticed that the class naming doesn't quite align with our decision to use PEP-8 Style [class naming convention](https://peps.python.org/pep-0008/#class-names). Just a friendly reminder for future reference!

Reply:
Thx for the heads up. I will fix them in a future PR.